packages/vite/src/node/optimizer/esbuildDepPlugin.ts

```diff
@@ -135,6 +136,24 @@ export function esbuildDepPlugin(
         }
       )
 
+      const rawUrlRE = /\?raw$/
+
+      build.onResolve(
+        { filter: rawUrlRE },
+        async ({ path: id, importer, kind }) => ({
+          path: await resolve(id.replace(rawUrlRE, ''), importer, kind),
+          namespace: 'raw'
+        })
+      )
+
+      build.onLoad({ filter: /.*/, namespace: 'raw' }, async ({ path }) => {
+        const text = await fs.promises.readFile(path, 'utf8')
+        return {
+          loader: 'json',
+          contents: JSON.stringify(text)
```

Comment:
**issue (if-minor):** Attention :warning:  

`JSON.stringify` returns a quoted string if a string was inserted. Is this wanted behavior here?

<img src="https://user-images.githubusercontent.com/7195563/115275958-bfa35d00-a142-11eb-890e-f55d6f0ad6c6.png" width="200px"/>

Reply:
> Is this wanted behavior here?
Yep